geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/planning/LocalQueryRunner.scala

```diff
@@ -282,27 +273,20 @@ object LocalQueryRunner {
     }
   }
 
-  private def arrowTransform(original: CloseableIterator[SimpleFeature],
-                             sft: SimpleFeatureType,
-                             transform: Option[(String, SimpleFeatureType)],
-                             hints: Hints,
-                             hook: Option[ArrowDictionaryHook]): CloseableIterator[SimpleFeature] = {
+  private def arrowTransform(
+      original: CloseableIterator[SimpleFeature],
+      sft: SimpleFeatureType,
+      transform: Option[(String, SimpleFeatureType)],
+      hints: Hints,
+      hook: Option[ArrowDictionaryHook]): CloseableIterator[SimpleFeature] = {
 
-    val sort = hints.getArrowSort
+    val sort = hints.getArrowSort.map(Seq.fill(1)(_))
```

Comment:
What's this trick?

Reply:
ah, not much of a trick... the arrow sort is defined as a single field (since we only support sorting on one field), while the general geomesa sort can handle a Seq of fields, so this just converts between the two.
i.e. we need to go from `Option[(String, Boolean)]` to `Option[Seq[(String, Boolean)]]`

Reply:
Ok, so if/when we refactor the Option[(String, Boolean)] to be Option[SortingClass], this would make more sense....